src/main/java/org/tdl/vireo/util/FileIOUtility.java

```diff
@@ -28,6 +32,9 @@
 
     @Autowired
     private ObjectMapper objectMapper;
+    
+	@Autowired
```

Comment:
We might want to start considering minimizing autowiring singletons in multiple services. Don't have a exact reason as of now, but it seems careless to just autowire singletons everywhere you happen to need them for the moment and consider a more architectural approach. Just some thoughts.